proto/xudrpc.proto

```diff
@@ -467,6 +467,15 @@ message Order {
   uint64 hold = 10 [json_name = "hold"];
 }
 
+message Currency {
+  // Ticker symbol of this currency (e.g. BTC)
+  string ticker_symbol = 1 [json_name = "ticker_symbol"];
+  // The places to the right of the decimal point of the smallest subunit (e.g. 8, 18)
+  uint32 decimal_places = 2 [json_name = "decimal_places"];
+  // Global identifier of this currency
```

Comment:
Can we drop the "global" term here in the comment as well?

Reply:
my bad